model/utils.go

```diff
@@ -20,6 +20,13 @@ import (
 	"github.com/pborman/uuid"
 )
 
+const (
+	LOWERCASE_LETTERS = "abcdefghijklmnopqrstuvwxyz"
+	UPPERCASE_LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	NUMBERS           = "0123456789"
+	SYMBOLS           = " !\"#$%&'()*+,-./:;<=>?@[]^_`|~"
```

Comment:
Actually I didn't want to use the backslash, because I did not know triple backslash was a thing